bitcoin-magazine/bitcoin-explained/silent-payments.md

```diff
@@ -0,0 +1,1229 @@
+---
+title: "Silent Payments"
+transcript_by: tijuan1 via review.btctranscripts.com
+media: https://www.youtube.com/watch?v=42PMLaz7Avk
+tags: ["silent-payments"]
+speakers: ["Sjors Provoost","Aaron Van Wirdum","Ruben Somsen"]
+categories: ["podcast"]
+date: 2022-06-09
+---
+
+## <Untitled Chapter 1>
```

Comment:
This is not needed. We've fixed that for newer transcripts - it was added automatically when the YouTube source doesn't have a chapter at the beginning of the video.